src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+public class Storage {
+    private static String filePath;
+    public Storage(String filePath){
+        Storage.filePath = filePath;
+    }
+    public static void appendToFile(String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter(Storage.filePath, true); // create a FileWriter in append mode
+        fw.write(textToAppend);
+        fw.close();
+    }
+    public static void preloadFromFile(TaskList list) {
```

Comment:
Very good idea to make a preloadFromFile function! I did not think of that. Yoinked